sql/dql/basic-queries/order-by-clause.md

```diff
@@ -41,117 +41,104 @@ aspects:
 ---
 ## Content
 
-`ORDER BY` clause enables users to order data in ascending or descending order on one or more columns. It comes after `GROUP BY` clause.
+The `ORDER BY` clause enables users to order data in ascending or descending order on one or more columns. This is also more commonly known as **sorting** data.
```

Comment:
> order data in ascending or descending order on one or more columns

might be worth mentioning these a bit later (after the first example)

you can start by just saying "The `ORDER BY` clause in SQL helps you sort data returned by your queries"

Reply:
changed entire paragraph to:

The `ORDER BY` clause in SQL helps you sort data returned by your queries.